tensorflow/core/kernels/matmul_op_test.cc

```diff
@@ -181,6 +181,9 @@ class FusedMatMulOpTest : public OpsTestBase {
                      .Attr("transpose_b", transpose_b)
                      .Finalize(&fused_matmul));
 
+#if TENSORFLOW_USE_ROCM
+    allow_gpu_device = false;
```

Comment:
Can we query some runtime property instead of using #if?

Reply:
@cheshire I have pushed new code that uses a runtime property. Please review.

Thanks!